_layouts/collections/item.html

```diff
@@ -44,32 +44,12 @@ <h2 id="{{ child.header | slugify }}">{{ child.header }}</h2>
 
   {% unless child.markup == "" %}
 
-  <div class="guide-example">
-    <h3 class="guide-example--heading">
-      <span>Rendered example:</span> {{ child.header }}
-    </h3>
-
-    <div class="guide-example--rendered">
-      {% include {{ child.markup }} %}
-    </div>
-
-    <h3 class="guide-example--heading">
-      <span><abbr>HTML</abbr>:</span> {{ child.header }}
-    </h3>
-
-    <div class="guide-example--code">
-      {% highlight html %}
-      {% include {{ child.markup }} %}
-      {% endhighlight %}
-    </div>
-
-    <cite class="guide-example--cite">
-      Stylesheet source on GitHub:
-      <a href="
-        {{ site.github_url }}/tree/master/{{ child.sourceFile.path }}#L{{ child.sourceFile.line }}"
-        rel="external"><code>{{ child.sourceFile.name }}</code></a>, line {{ child.sourceFile.line }}
-    </cite>
-  </div>
+  {% include guide_example.html
```

Comment:
Actually, @petronbot what do you think about renaming this include to `guide_example_markup.html`? I could see the possibility of trying to include other code examples, eg SASS code?